frozen-abi/src/abi_example.rs

```diff
@@ -561,3 +561,12 @@ impl<T: AbiExample> AbiExample for once_cell::sync::OnceCell<T> {
         Self::with_value(T::example())
     }
 }
+
+use crossbeam_channel::{unbounded, Sender};
+
+impl<T: AbiExample> AbiExample for Sender<T> {
```

Comment:
what change made this a requirement?  a channel should definitely _not_ be part of ABI